source/core_mqtt.c

```diff
@@ -1002,13 +1002,14 @@ static MQTTStatus_t sendPublishAcks( MQTTContext_t * pContext,
 static MQTTStatus_t handleKeepAlive( MQTTContext_t * pContext )
 {
     MQTTStatus_t status = MQTTSuccess;
-    uint32_t now = 0U, keepAliveMs = 0U;
+    uint32_t now = 0U, keepAliveMs = 0U, packetRxTimeoutMs = 0U;
 
     assert( pContext != NULL );
     assert( pContext->getTime != NULL );
 
     now = pContext->getTime();
     keepAliveMs = 1000U * ( uint32_t ) pContext->keepAliveIntervalSec;
+    packetRxTimeoutMs = 1000U * ( uint32_t ) pContext->packetRxTimeoutSec;
```

Comment:
Can we use the same parameter as before for determining when to send a keep alive?